tests/src/dir.targets

```diff
@@ -75,6 +75,11 @@
     <MSBuild Projects="@(ProjectReference)" />
     <MakeDir Condition="'$(CLRTestKind)' == 'RunOnly'" ContinueOnError="false" Directories="$(OutputPath)" />
   </Target>
+ 
+  <Target Name="CopyJsonProjectFiles" AfterTargets="Build">  
+    <!-- Post build copy project json files so we can generate assembly lists for the projects -->  
+    <Copy SourceFiles="$(ProjectJson);$(ProjectLockJson)" DestinationFolder="$(OutputPath)" />  
```

Comment:
I would be OK if you did this only for perf tests but since this happens for all the tests. This would add up.